Contributing-Guidelines.md

```diff
@@ -0,0 +1,35 @@
+# Contributing Guidelines
+
+To learn more about the purpose of TEPs and how to go about writing one, please start reading at TEP 1. Also, make sure to check the [README](/README.md) for information on how to render the TEPs in this repository.
+
+Thanks again for your contributions, and we look forward to reviewing them!
+
+## Before writing a new TEP
+
+Prior to submitting a pull request here with your draft TEP, see TEP 1 for some important steps to consider, including proposing and discussing it first in an appropriate venue, drafting a TEP and gathering feedback, and developing at least a prototype reference implementation of your idea.
+
+## Contributing changes to existing TEPs
+
+In general, most non-Draft/Active TEPs are considered to be historical documents rather than living specifications or documentation. Major changes to their core content usually require a new TEP, while smaller modifications may or may not be appropriate, depending on the TEP's status.
+
+See TEP Maintenance and Changing Existing TEPs in TEP 1 for more.
+
+Copyediting and proofreading Draft and Active TEPs is welcome (subject to review by the TEP author), and can be done via pull request to this repo. Substantive content changes should first be proposed on TEP discussion threads. We do advise against PRs that simply mass-correct minor typos on older TEPs which don't significantly impair meaning and understanding.
+
+If you're still unsure, we encourage you to reach out first before opening a PR here. For example, you could contact the TEP author(s), propose your idea in a discussion venue appropriate to the TEP (such as Typing-SIG for static typing, or Packaging Discourse for packaging), or open an issue.
+
+## Commit messages and PR titles
+
+When adding or modifying a TEP, please include the TEP number in the commit summary and pull request title. For example, "TEP NNN: summary of changes".
+
+Likewise, prefix rendering infrastructure changes with Infra:, linting alterations with Lint: and other non-TEP meta changes, such as updates to the Readme/Contributing Guide, issue/PR template, etc., with Meta:.
+
+## Sign the CLA
+
+Before you hit "Create pull request", please take a moment to ensure that this project can legally accept your contribution by verifying you have signed the [Contributor License Agreement](https://github.com/torrust/.github/blob/main/info/licensing/contributor_agreement_v01.md).
+
+If you haven't signed the CLA before, please follow the steps outlined in the CPython devguide to do so.
```

Comment:
I think we need to include instructions for signing the CLA instead of pointing to the CPython devguide.

Reply:
Correct. This slipped through it.

Reply:
Done